pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerStatsRecorderImpl.java

```diff
@@ -218,44 +224,63 @@ void updateCumulativeStats(ProducerStats stats) {
         totalBytesSent.add(stats.getNumBytesSent());
         totalSendFailed.add(stats.getNumSendFailed());
         totalAcksReceived.add(stats.getNumAcksReceived());
+        sendMsgsRate += stats.getSendMsgsRate();
+        sendBytesRate += stats.getSendBytesRate();
+        if (latencyPctValues == null || latencyPctValues.length == 0) {
+            latencyPctValues = stats.getLatencyPctValues();
+        } else {
+            double[] curLatencyPctValues = stats.getLatencyPctValues();
```

Comment:
it maybe not right here.
e.g. if a topic have 2 partitions, with med latency 3ms and 5ms, It does not mean this topic has a med latency 8 ms. 
This metric seems hard to get cumulative-ed.

Reply:
Yes, seems that the latency for each partition should be collection , then the total delay calculated based on the quantile.

Reply:
Different calibers may confuse users